hass_nabucasa/__init__.py

```diff
@@ -174,9 +174,9 @@ async def fetch_subscription_info(self):
 
     async def login(self, email: str, password: str) -> None:
         """Log a user in."""
-        with async_timeout.timeout(15):
+        async with async_timeout.timeout(30):
```

Comment:
Here you say 30s, description says 60s.

Reply:
You are right :) It was the first 60s but the limit of lambda is 30s, so I was not sure if there is a general timeout around 30s for AWS services. However, what you want to have a timeout?

Reply:
let's start with 30.